src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -288,397 +185,419 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
+          }
         }
       }
     }
 
 
     "Passthrough tests on the /workspaces/%s/%s path" - {
-      "MethodNotAllowed error is not returned for HTTP GET and DELETE methods" in {
-        List(HttpMethods.GET, HttpMethods.DELETE) map {
-          method =>
-            new RequestBuilder(method)(workspacesRoot) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
-              status shouldNot equal(MethodNotAllowed)
-            }
+      List(HttpMethods.GET, HttpMethods.DELETE) foreach { method =>
+        s"OK status is returned for HTTP $method" in {
+          stubRawlsService(method, workspacesPath, OK)
+          new RequestBuilder(method)(workspacesPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
+            status should equal(OK)
+          }
         }
       }
     }
 
 
     "Passthrough tests on the /workspaces/%s/%s/methodconfigs path" - {
-      "MethodNotAllowed error is not returned for HTTP GET and POST methods" in {
-        List(HttpMethods.GET, HttpMethods.POST) map {
-          method =>
-            new RequestBuilder(method)(methodconfigsPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
-              status shouldNot equal(MethodNotAllowed)
-            }
+      List(HttpMethods.GET, HttpMethods.POST) foreach { method =>
+        s"OK status is returned for HTTP $method" in {
+          stubRawlsService(method, methodconfigsPath, OK)
+          new RequestBuilder(method)(methodconfigsPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
+            status should equal(OK)
+          }
         }
       }
     }
 
 
     "Passthrough tests on the /workspaces/%s/%s/updateAttributes path" - {
-      "MethodNotAllowed error is not returned for HTTP PATCH method" in {
-        Patch(updateAttributesPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
-          status shouldNot equal(MethodNotAllowed)
+      "OK status is returned for HTTP PATCH" in {
+        // Careful here... although this is a passthrouth, orchestration does not mirror the same URL as rawls in this case
+        stubRawlsService(HttpMethods.PATCH, workspacesPath, OK)
+        Patch(updateAttributesPath, "[]") ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
+          status should equal(OK)
         }
       }
     }
 
 
     "Passthrough tests on the /workspaces/%s/%s/acl path" - {
-      "MethodNotAllowed error is not returned for HTTP GET and PATCH methods" in {
-        List(HttpMethods.GET, HttpMethods.PATCH) map {
-          method =>
-            new RequestBuilder(method)(aclPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
-              status shouldNot equal(MethodNotAllowed)
-            }
-        }
-      }
-    }
-
-
-    "Passthrough tests on the /workspaces/%s/%s/clone path" - {
-      "MethodNotAllowed error is not returned for POST method" in {
```

Comment:
POST on /clone is not actually a passthrough, so the test has been moved to a new section below.